src/aiida/tools/pytest_fixtures/storage.py

```diff
@@ -52,6 +53,8 @@ def create_database(
         cluster = PGTest()
         cluster.create_database = create_database
         yield cluster
+    except OSError:
```

Comment:
This is what I am getting (don't have Postgres globally installed

```python
 OSError: b'initdb: error: file "/opt/conda/envs/aiida-core-services/share/postgres.bki" does not exist\ninitdb: hint: This might mean you have a corrupted installation or identified the wrong directory with the invocation option -L.\npg_ctl: database system initialization failed\n'
```